lib/CL/clCreateBuiltinKernelWithAttributes.cc

```diff
@@ -0,0 +1,480 @@
+// OpenCL runtime library: clCreateBuiltinKernelWithAttributesEXP()
```

Comment:
Please make this a C file. We plan to launch DBKs from very low end edge devices which might have very limited OS/system libs and memory.